sdk/src/client/storage/mod.rs

```diff
@@ -18,15 +18,15 @@ pub trait StorageProvider {
     type Error;
 
     /// Get a value out of the database.
-    async fn get(&mut self, k: &[u8]) -> Result<Option<Vec<u8>>, Self::Error>;
```

Comment:
Changelog

Reply:
It's not public?

Reply:
It's a public trait, you say it's not publicly exported?